### PR TITLE
Pioneer 13 NUTNR predeploy cal csvs

### PR DESCRIPTION
**Pull request by rtravis-whoi**
Included in this PR are 3 new NUTNR cal csvs created from Pioneer 13 pre-deployment testing.

### CODE REVIEW

File: calibration/NUTNRB/CGINS-NUTNRB-01106__20190923.csv [hunk 1]
```diff
@@ -0,0 +1,8 @@
...
...8, 273.28, 274.09, 274.89, 275.7, 276.5, 277.31, 278.11, 278.92, 279.72, 280.53, 281.33, 282.14, 282.94, 283.75, 284.56, 285.36, 286.17, 286.97, 287.78, 288.58, 289.39, 290.2, 291, 291.81, 292.61, 293.42, 294.22, 295.03, 295.84, 296.64, 297.45, 298.25, 299.06, 299.86, 300.67, 301.48, 302.28, 303.09, 303.89, 304.7, 305.5, 306.31, 307.12, 307.92, 308.73, 309.53, 310.34, 311.14, 311.95, 312.75, 313.56, 314.36, 315.17, 315.97, 316.78, 317.58, 318.39, 319.19, 320, 320.8, 321.61, 322.41, 323.22, 324.02, 324.83, 325.63, 326.44, 327.24, 328.04, 328.85, 329.65, 330.46, 331.26, 332.06, 332.87, 333.67, 334.48, 335.28, 336.08, 336.89, 337.69, 338.49, 339.3, 340.1, 340.9, 341.7, 342.51, 343.31, 344.11, 344.91, 345.72, 346.52, 347.32, 348.12, 348.92, 349.72, 350.53, 351.33, 352.13, 352.93, 353.73, 354.53, 355.33, 356.13, 356.93, 357.73, 358.53, 359.33, 360.13, 360.93, 361.73, 362.53, 363.33, 364.13, 364.93, 365.73, 366.53, 367.32, 368.12, 368.92, 369.72, 370.52, 371.31, 372.11, 372.91, 373.71, 374.5, 375.3, 376.1, 376.89, 377.69, 378.48, 379.28, 380.08, 380.87, 381.67, 382.46, 383.26, 384.05, 384.84, 385.64, 386.43, 387.23, 388.02, 388.81, 389.61, 390.4, 391.19, 391.98, 392.78, 393.57, 394.36]",
```
Comment (snwhite) [inline calibration/NUTNRB/CGINS-NUTNRB-01106__20190923.csv]: Looks good.  Spot checked from SNA1006C.CAL

File: calibration/NUTNRB/CGINS-NUTNRB-01062__20190923.csv [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
...272.74, 273.55, 274.37, 275.18, 275.99, 276.81, 277.62, 278.43, 279.25, 280.06, 280.87, 281.69, 282.5, 283.31, 284.13, 284.94, 285.75, 286.57, 287.38, 288.2, 289.01, 289.82, 290.64, 291.45, 292.26, 293.08, 293.89, 294.7, 295.52, 296.33, 297.14, 297.96, 298.77, 299.58, 300.4, 301.21, 302.02, 302.84, 303.65, 304.46, 305.28, 306.09, 306.9, 307.71, 308.53, 309.34, 310.15, 310.97, 311.78, 312.59, 313.4, 314.22, 315.03, 315.84, 316.65, 317.46, 318.28, 319.09, 319.9, 320.71, 321.52, 322.34, 323.15, 323.96, 324.77, 325.58, 326.39, 327.2, 328.02, 328.83, 329.64, 330.45, 331.26, 332.07, 332.88, 333.69, 334.5, 335.31, 336.12, 336.93, 337.74, 338.55, 339.36, 340.17, 340.98, 341.79, 342.6, 343.41, 344.21, 345.02, 345.83, 346.64, 347.45, 348.26, 349.06, 349.87, 350.68, 351.49, 352.3, 353.1, 353.91, 354.72, 355.52, 356.33, 357.14, 357.94, 358.75, 359.55, 360.36, 361.17, 361.97, 362.78, 363.58, 364.39, 365.19, 366, 366.8, 367.6, 368.41, 369.21, 370.02, 370.82, 371.62, 372.42, 373.23, 374.03, 374.83, 375.63, 376.44, 377.24, 378.04, 378.84, 379.64, 380.44, 381.24, 382.04, 382.84, 383.64, 384.44, 385.24, 386.04, 386.84, 387.64, 388.44, 389.24, 390.04, 390.83, 391.63, 392.43, 393.23, 394.02, 394.82]",
```
Comment (snwhite) [inline calibration/NUTNRB/CGINS-NUTNRB-01062__20190923.csv]: Looks good.  Spot checked against SNA1062C.CAL.

File: calibration/NUTNRB/CGINS-NUTNRB-01063__20190923.csv [hunk 3]
```diff
@@ -0,0 +1,8 @@
...
...28, 275.09, 275.9, 276.72, 277.53, 278.34, 279.15, 279.96, 280.77, 281.58, 282.39, 283.2, 284.01, 284.82, 285.63, 286.44, 287.26, 288.07, 288.88, 289.69, 290.5, 291.31, 292.12, 292.93, 293.74, 294.55, 295.36, 296.17, 296.98, 297.79, 298.6, 299.41, 300.22, 301.03, 301.84, 302.65, 303.46, 304.27, 305.08, 305.89, 306.7, 307.51, 308.31, 309.12, 309.93, 310.74, 311.55, 312.36, 313.17, 313.98, 314.78, 315.59, 316.4, 317.21, 318.02, 318.83, 319.63, 320.44, 321.25, 322.06, 322.86, 323.67, 324.48, 325.29, 326.09, 326.9, 327.71, 328.51, 329.32, 330.12, 330.93, 331.74, 332.54, 333.35, 334.15, 334.96, 335.77, 336.57, 337.38, 338.18, 338.99, 339.79, 340.59, 341.4, 342.2, 343.01, 343.81, 344.61, 345.42, 346.22, 347.02, 347.83, 348.63, 349.43, 350.23, 351.04, 351.84, 352.64, 353.44, 354.24, 355.05, 355.85, 356.65, 357.45, 358.25, 359.05, 359.85, 360.65, 361.45, 362.25, 363.05, 363.85, 364.65, 365.45, 366.24, 367.04, 367.84, 368.64, 369.44, 370.23, 371.03, 371.83, 372.62, 373.42, 374.22, 375.01, 375.81, 376.6, 377.4, 378.2, 378.99, 379.78, 380.58, 381.37, 382.17, 382.96, 383.75, 384.55, 385.34, 386.13, 386.93, 387.72, 388.51, 389.3, 390.09, 390.88, 391.67, 392.46, 393.25, 394.04, 394.83, 395.62]",
```
Comment (snwhite) [inline calibration/NUTNRB/CGINS-NUTNRB-01063__20190923.csv]: Looks good.  Spot checked against SNA1063C.CAL.

File: calibration/NUTNRB/CGINS-NUTNRB-01106__20190923.csv (hunk 1 above)
Comment (reedan88) [reply calibration/NUTNRB/CGINS-NUTNRB-01106__20190923.csv]: Looks good.

**Pull request merged**